homeassistant/components/mqtt/device_tracker.py

```diff
@@ -5,30 +5,55 @@
 
 from homeassistant.components import mqtt
 from homeassistant.components.device_tracker import PLATFORM_SCHEMA
+from homeassistant.components.device_tracker.const import (
+    SOURCE_TYPE_GPS,
+    SOURCE_TYPE_ROUTER,
+    SOURCE_TYPE_BLUETOOTH,
+    SOURCE_TYPE_BLUETOOTH_LE,
+)
```

Comment:
We should create a `SOURCE_TYPE_ALL`  on `device_tracker.const` as a list and import that list. So we don't need to care about future changes.